config/application.rb

```diff
@@ -54,7 +54,7 @@ class Application < Rails::Application
       end
     end
 
-    config.load_defaults '7.0'
+    config.load_defaults '7.1'
```

Comment:
Is there documentation somewhere for the effect of this change? i.e. what are the new defaults?

Reply:
https://guides.rubyonrails.org/configuring.html#versioned-default-values